homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -54,12 +55,14 @@
 DEFAULT_PORT = 80
 DEFAULT_PASSWORD = "1234"
 DEVICE_URL = "http://{0}:{1}/device"
+DEFAULT_NAME = ""
```

Comment:
Please specify a default name e.g. `Frontier Silicon`

Reply:
Hi, my reason for not specifying a hard-coded default was that if the user doesn't manually configure a name it will fallback to the existing behaviour (which is to query the radio for its name).  I feel this is highly preferable.  Also it would make it a breaking change as any existing users would find all their radios renamed to the new default.  Would you agree?

Reply:
Yes I agree with that. Please remove the constant and the usage of the constant then.

Reply:
Done.